inference-engine/tests/functional/plugin/shared/src/hetero/synthetic.cpp

```diff
@@ -106,6 +106,7 @@ void HeteroSyntheticTest::SetUp() {
         --num;
     }
     function = std::get<Function>(param)._function;
+    functionRefs = function;
```

Comment:
use 'ngraph::clone_function(function);'  instead of '='

Reply:
Fixed, thanks